diff --git a/csslint/csslint.c b/csslint/csslint.c
index faf4c09..de92f44 100644
--- a/csslint/csslint.c
+++ b/csslint/csslint.c
@@ -25,6 +25,7 @@
 
 #include "libcroco.h"
 #include <libxml/xpath.h>
+#include <libxml/parser.h>
 
 #include <glib.h>
 #include <string.h>
diff --git a/tests/test5-main.c b/tests/test5-main.c
index 22b9911..a77a045 100644
--- a/tests/test5-main.c
+++ b/tests/test5-main.c
@@ -21,6 +21,7 @@
  */
 
 #include <string.h>
+#include <libxml/parser.h>
 #include "cr-test-utils.h"
 #include "libcroco.h"
 
